### PR TITLE
Connectivity in Device details

### PR DESCRIPTION
**Pull request by MWindel**
- Added connectivity to device details table in device management
- Added link to Device management library in device details

### CODE REVIEW

**Review by rssag — DISMISSED** [review]

File: content/users-guide/device-management-bundle/device-details.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ To view detailed information on the device, click a device in the device list.
 
 ![Device info](/images/users-guide/DeviceManagement/devmgmt-devices-info.png)
 
-The device details are divided into tabs. The number of tabs is dynamic and depends on the available information, i.e. tabs are only displayed if the kind of information is available for the particular device.
+The device details are divided into tabs. The number of tabs is dynamic and depends on the available information, i.e. tabs are only displayed if the kind of information is available for the particular device. For a detailed description of the operations and fragments related to each device detail tab see the [Device Management Library](/reference/device-management-library/#overview) in the *Reference guide*.
```
Comment (BeateRixen) [inline content/users-guide/device-management-bundle/device-details.md:12]: Device Management Library > Device management library

File: content/users-guide/device-management-bundle/device-details.md [hunk 2]
```diff
@@ -42,14 +42,14 @@ The following tabs are the most common ones, each described in detail in a separ
...
-<td style="text-align:left"><a href="../../protocol-integration/cloud-fieldbus">Fieldbus</a></td>
-<td style="text-align:left">Provides details for fieldbus devices. For details, see <a href="../../protocol-integration/cloud-fieldbus">Cloud Fieldbus</a>.</td>
-</tr>
-<tr>
 <td align="left"><a href="#config">Configuration</a></td>
 <td align="left">Allows manual configuration of device parameters and settings entered in a text format. For details, see <a href="#configuration-repository">Managing configurations</a> for binary configuration.</td>
 </tr>
 <tr>
+<td style="text-align:left"><a href="">Connectivity</a></td>
+<td style="text-align:left">Provides SIM management funktionality. For details, see <a href="../../reference/device-management-library/#connectivity">Connectivity</a> in the Device Management Library.</td>
```
Comment (BeateRixen) [inline content/users-guide/device-management-bundle/device-details.md:50]: funktionality > functionality
For details, I would better link to the Connectivity chapter https://cumulocity.com/guides/users-guide/device-management/#connectivity

File: content/users-guide/device-management-bundle/device-details.md (hunk 1 above)
Comment (MWindel) [reply content/users-guide/device-management-bundle/device-details.md:12]: Done

File: content/users-guide/device-management-bundle/device-details.md (hunk 2 above)
Comment (MWindel) [reply content/users-guide/device-management-bundle/device-details.md:50]: Done

**Review by BeateRixen — APPROVED** [review]

**Review by rssag — APPROVED** [review]

**Comment by BeateRixen** [comment]
@MWindel Do not cherry-pick for now, as the device management library is not yet available. This needs to be cherry-picked after the library PR is merged.

**Pull request merged**